keylime.conf

```diff
@@ -110,6 +113,11 @@ revocation_actions=
 # with a working directory of /var/lib/keylime/secure/unzipped.
 payload_script=autorun.sh
 
+# In case mTLS for the agent is disabled and the use of payloads is still
+# required, this option has to be set to "True" in order to allow the agent 
+# to start.
+enable_insecure_payload = False
```

Comment:
Add a reference to https://github.com/keylime/keylime/security/advisories/GHSA-2m39-75g9-ff5r and explain why this is generally a bad idea to do.

Reply:
done